openbb_terminal/stocks/fundamental_analysis/polygon_model.py

```diff
@@ -0,0 +1,104 @@
+"""Polygon Model"""
+__docformat__ = "numpy"
+
+import logging
+from typing import Tuple
+
+import pandas as pd
+import requests
+
+from openbb_terminal import config_terminal as cfg
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+
+@log_start_end(log=logger)
+def get_financials(
```

Comment:
Can we just return either income or balance based on a selection here instead of returning multiple data frames?

It makes it easier for when to use API

Reply:
So I was gonna do this, but given this has a free tier limit of 5/minute I thought we could figure out a way to cache both so you don't have to do twice the calls.